src/c++/perf_analyzer/genai-perf/tests/test_csv_exporter.py

```diff
@@ -168,7 +179,13 @@ def test_nonstreaming_llm_csv_output(
             "Output Token Throughput (per sec),456.00\r\n",
             "Request Throughput (per sec),123.00\r\n",
         ]
-        returned_data = mock_read_write
+        returned_data = [
+            data for filename, data in mock_read_write if filename == expected_filename
+        ]
```

Comment:
do we get multiple csv files? (I thought we only generate a single csv file)

Reply:
We only generate one CSV, but we write is called multiple time. It's a list of objects (e.g. CSV rows).